### PR TITLE
Add Thunk middleware endpoint

### PR DESCRIPTION
**Pull request by matiasgarcia91**

### CODE REVIEW

File: src/store/posts/actions.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import axios from "axios";
 const API_URL = `https://codaisseur-coders-network.herokuapp.com/posts`;
 
-export const savePosts = posts => ({
+export const savePosts = (posts) => ({
```
Comment (matiasgarcia91) [inline src/store/posts/actions.js:4]: Why add parenthesis? I like it without

File: src/store/posts/actions.js [hunk 2]
```diff
@@ -48,15 +48,15 @@ export const fetchPostById = id => async (dispatch, getState) => {
...
-//     action => {
-//       if (typeof action === "function") {
-//         // action => fetch5Posts
-//         return action(dispatch, getState, extraArgument); // function actions => RUN.
-//       }
-
-//       return next(action); // non function actions => next => to the store!
-//     };
-// }
+function createThunkMiddleware(extraArgument) {
```
Comment (leahcarlin) [inline src/store/posts/actions.js:51]: Prefer to have this commented out for now

File: src/store/posts/actions.js (hunk 1 above)
Comment (rlcngz) [reply src/store/posts/actions.js:4]: Prettier is organising it automatically

**Review by AlessandroGenerale17 — APPROVED** [review]
🇮🇹

**Review by mgl-aguiar — APPROVED** [review]

**Pull request merged**